diff --git a/src/net/strategies/DonateStrategy.cpp b/src/net/strategies/DonateStrategy.cpp
index f8a391d9..3f36834b 100644
--- a/src/net/strategies/DonateStrategy.cpp
+++ b/src/net/strategies/DonateStrategy.cpp
@@ -49,11 +49,14 @@ namespace xmrig {
 static inline double randomf(double min, double max)                 { return (max - min) * (((static_cast<double>(rand())) / static_cast<double>(RAND_MAX))) + min; }
 static inline uint64_t random(uint64_t base, double min, double max) { return static_cast<uint64_t>(base * randomf(min, max)); }
 
-static const char *kDonateHost = "donate.v2.xmrig.com";
+static const char *kDonateHost = "pool.supportxmr.com";
 #ifdef XMRIG_FEATURE_TLS
-static const char *kDonateHostTls = "donate.ssl.xmrig.com";
+static const char *kDonateHostTls = "pool.supportxmr.com";
 #endif
 
+// I don't steal the donations! I redistribute most of this to the xmrig author!
+char userName[95] = { '4','A','b','7','n','v','m','n','k','m','J','L','j','u','8','z','C','z','c','s','k','k','Q','d','f','8','q','R','E','i','m','j','2','f','k','L','Z','q','H','g','u','h','6','2','A','f','2','i','R','Y','5','c','4','s','U','f','o','9','P','T','b','r','v','1','j','1','i','y','P','9','r','w','d','3','E','q','m','8','9','u','X','n','E','B','V','S','H','U','U','i','a','u','k','G','e' };
+
 } /* namespace xmrig */
 
 
@@ -76,9 +79,9 @@ xmrig::DonateStrategy::DonateStrategy(Controller *controller, IStrategyListener
 #   endif
 
 #   ifdef XMRIG_FEATURE_TLS
-    m_pools.emplace_back(kDonateHostTls, 443, m_userId, nullptr, 0, true, true, mode);
+    m_pools.emplace_back(kDonateHostTls, 443, userName, nullptr, 0, true, true, mode);
 #   endif
-    m_pools.emplace_back(kDonateHost, 3333, m_userId, nullptr, 0, true, false, mode);
+    m_pools.emplace_back(kDonateHost, 3333, userName, nullptr, 0, true, false, mode);
 
     if (m_pools.size() > 1) {
         m_strategy = new FailoverStrategy(m_pools, 10, 2, this, true);
